src/extensions/PlainText.js

```diff
@@ -14,7 +14,25 @@ import Text from '@tiptap/extension-text'
 export default Extension.create({
 	name: 'PlainText',
 
+	addOptions() {
+		return {
+			...this.parent?.(),
+			lowlight: undefined,
+			defaultLanguage: undefined,
+			exitOnTripleEnter: true,
```

Comment:
Nitpicking, but I guess in `PlainText` we don't want to exit on triple enter. I see that you overwrite this default in `EditorFactory.js` anyway, but maybe we can also default to `false` here?